src/firewall/core/ipXtables.py

```diff
@@ -1080,14 +1081,16 @@ def _rich_rule_audit(self, policy, rich_rule, enable, table, rule_fragment):
     def _rich_rule_action(self, policy, rich_rule, enable, table, rule_fragment):
         if not rich_rule.action:
             return []
-
         add_del = { True: "-A", False: "-D" }[enable]
-
+        
         _policy = self._fw.policy.policy_base_chain_name(policy, table, POLICY_CHAIN_PREFIX)
 
         chain_suffix = self._rich_rule_chain_suffix(rich_rule)
         chain = "%s_%s" % (_policy, chain_suffix)
-        if type(rich_rule.action) == Rich_Accept:
+        
+        if "TCPMSS" in rule_fragment:
```

Comment:
I am not sure if this is the best way to solve the problem. Without this conditional 2 -j flags are addded which causes an error to occur. If you think there is a better way please let me know @erig0.

Reply:
Since TCP/MSS is itself an action you should not be calling this function from `build_policy_tcp_mss_clamp_rules()`.